### PR TITLE
chore: setup software environment

### PR DESCRIPTION
**Pull request by tianrui-wei**
Signed-off-by: Tianrui Wei <tianrui@tianruiwei.com>

*Issue #, if available:*

None

*Description of changes:*

This PR completes the setup instructions provided, and corrects a misalignment between commands.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,16 +48,28 @@ The customers will clone the dev kit using
 They can also simply download the dev kit provided as tar ball from xilinx Alveo Vivado flow [webpage](https://www.xilinx.com/products/boards-and-kits/alveo/u200.html#gettingStarted).
 
 *STEP2*:
+
+setup the software development environment
+
+	source sdk_setup.sh
+	cd sdk
```
Comment (kyyalama2) [inline README.md]: Just source sdk_setup.sh is sufficient. We do not need to manually do sdk_install.sh. That should be automatically taken care of with source sdk_setup.sh

File: README.md [hunk 2]
```diff
@@ -48,16 +48,28 @@ The customers will clone the dev kit using
...
+	sudo ./sdk_install.sh
+
+*STEP3*:
+
+Install the patched xdma driver according to instructions in $ROOT/sdk/xdma_patch/xdma_install.md
```
Comment (kyyalama2) [inline README.md:58]: This is only required for cl_dram_dma example or customer designs that use DMA.

File: README.md (hunk 1 above)
Comment (tianrui-wei) [reply README.md]: Thanks for the info. This issue has been resolved by the new commit.

File: README.md (hunk 2 above)
Comment (tianrui-wei) [reply README.md:58]: I pushed a new commit, marking this step as optional.

**Pull request merged**